store/type-map.js

```diff
@@ -1366,7 +1366,7 @@ export const mutations = {
 
   // setGroupDefaultType({group: 'core', defaultType: 'name'});
   // By default when a group is clicked, the first item is selected - this allows
-  // this behvaiour to be changed and a named child type can be chosen
+  // this behaviour to be changed and a named child type can be chosen
```

Comment:
For changes this large, I don't enforce en-* variations. But as the sponsor is SUSE, I'd assume en-GB is the preferred flavor.

Reply:
Your reasoning is sound. I've always left it as en-GB when written as such. We do tend to weave back and forth depending on author 🙂

@richard-cox do we have a preference that we enforce, or is it truly up to the author?

Reply:
The documentation uses American spelling. For a while it was kind of Western US themed and said things like "Howdy pardner."

Reply:
Thanks @catherineluse 🙂

I'm not perturbed about the variation for this one. Seeing as how it's a code comment, I'm a little inclined to just leave it as `en-gb`.

Reply:
If the formal style is `en-US`, I'm happy to make a distinct PR for the repo (or repos) to switch. cowboy style language is fine, but I won't cowboy it, it'd go in as a distinct PR.

Reply:
As much as it pains me to say... `en-US` should be used everywhere. It's not critical that we fix existing text though